src/components/MissionsList.jsx

```diff
@@ -0,0 +1,24 @@
+import { useSelector } from 'react-redux';
+
+const MissionsList = () => {
+  const { missions } = useSelector((store) => store.missions);
+
+  return (
+    <>
+      {missions.map((mission) => (
+        <tr key={mission.mission_id}>
+          <td>{mission.mission_name}</td>
+          <td>{mission.description}</td>
+          <td>status</td>
+          <td>
+            <button type="button">
+              Join mission
+            </button>
+          </td>
+        </tr>
+      ))}
+    </>
```

Comment:
Make sure to close the whole data into the biggest _container_ It will be helpful for you in your _**Styling**_ time and also get rid of overflow-x.

Reply:
Yes, you are right but the biggest container is MissionsTable.jsx where MissionList.jsx is imported so it won't need an extra container.